v10.x/bootstrap

```diff
@@ -1,5 +1,6 @@
 #!/bin/dash -e
 
+NODE_PATH="/opt/nodejs/node8/node_modules/:/opt/nodejs/node_modules:$LAMBDA_RUNTIME_DIR/node_modules" \
```

Comment:
Should this not be `export NODE_PATH=...`?

Reply:
Good point. Not sure there's a functional difference in this case? I'm passing it into Node on the command line directly and that's the only command in the script. Maybe cleaner though to export it?

Reply:
Ah, I missed the `\` at the end of the line, my bad

Reply:
No worries!